columnflow/tasks/framework/parameters.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
+
+"""
+Custom luigi parameters.
+"""
+
+from typing import Union
+
+import law
+from columnflow.util import test_float
+
+
+class SettingsParameterBase():
+    """
+    Base class to implement class methods for different types of setting parameters.
+    """
+    @classmethod
+    def parse_setting(self, setting: str) -> tuple[str, Union[float, bool, str]]:
```

Comment:
and `tuple[str, float | bool | str]` should work here.

Reply:
seems to not work here, not sure why